README.md

```diff
@@ -112,7 +112,7 @@ npm install @mui/material @mui/styled-engine-sc styled-components
 yarn add @mui/material @mui/styled-engine-sc styled-components
 ```
 
-Visit our [`styled-engine` guide](https://mui.com/material-ui/guides/styled-engine/) for more information about how to configure `styled-components` as the style engine.
+Visit our [styled-components guide](https://mui.com/material-ui/guides/styled-components/) for more information on configuration.
```

Comment:
I think that the true location of this guide should be here:

```suggestion
Visit our [styled-components guide](https://mui.com/system/guides/styled-components/) for more information on configuration.
```

But we could say it's beyond the scope of this PR.